### ISSUE TITLE
Fragment switch

### ISSUE THREAD

**Issue by aat-antoine** [lapism/SearchView#187]
In one activity I can switch Fragment by clicking on item from toolbar. I have two fragments : first with a map, second with list of obj and **SearchView** component. Ws is made to populate **SearchAdapter**. Sometimes the following crash appears.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'android.database.sqlite.SQLiteDatabase android.content.Context.openOrCreateDatabase(java.lang.String, int, android.database.sqlite.SQLiteDatabase$CursorFactory, android.database.DatabaseErrorHandler)' on a null object reference
        at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:223)
        at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:163)
        at com.lapism.searchview.SearchHistoryTable.open(SearchHistoryTable.java:29)
        at com.lapism.searchview.SearchHistoryTable.getAllItems(SearchHistoryTable.java:116)
        at com.lapism.searchview.SearchAdapter$1.publishResults(SearchAdapter.java:115)
        at android.widget.Filter$ResultsHandler.handleMessage(Filter.java:282)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:155)
        at android.app.ActivityThread.main(ActivityThread.java:5696)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1028)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:823)
```

**Comment by lapism** [comment]
Looks like a bug in database. I will fix it ASAP.

**Comment by lapism** [comment]
It will be fixed in 4.0.
